packages/manager/apps/catalog-revamp/src/components/Filters/Filters.scss

```diff
@@ -0,0 +1,54 @@
+.filters-container {
+  position: relative;
+  overflow: visible;
+  display: flex;
+  margin: 3em;
+  float: left;
```

Comment:
Can we avoid using float also ?
We have flexbox and grid, we don't need float anymore

Reply:
Done using tailwind now